src/test/scala/nl.knaw.dans.easy.bagstore/server/StoresServletSpec.scala

```diff
@@ -400,40 +412,76 @@ class StoresServletSpec extends TestSupportFixture
     retrieveBag(uuid3, "c")
   }
 
+  it should "fail when no BasicAuth credentials are provided" in {
+    val uuid = "11111111-1111-1111-1111-111111111111"
```

Comment:
Why not `UUID.randomUUID().toString`?

Reply:
Good question!

This is because that's how we do it everywhere in this testsuite. We're using the test resources which have specific UUIDs rather than random ones.